package.json

```diff
@@ -15,12 +15,10 @@
   "engines": {
     "node": ">=16.x"
   },
-  "browserslist": "> 0.5%, last 2 versions, Firefox ESR, not dead, ie >= 11",
   "dependencies": {
-    "@babel/core": "7.16.0",
+    "@babel/core": "7.17.8",
```

Comment:
How is babel used? Is it still needed here?

Reply:
Babel is still used by jest, I haven't fully explored possible options to trimming it, but I'd like to (probably in a separate PR).